domain/network/state/subnet.go

```diff
@@ -329,7 +341,11 @@ WHERE  subnet_uuid = $M.id;`
 		return nil, errors.Trace(err)
 	}
 
-	return &rows.ToSubnetInfos()[0], nil
+	infos, err := rows.ToSubnetInfos()
+	if err != nil {
+		return nil, errors.Annotate(err, "reconstructing subnet info from rows")
+	}
+	return &infos[0], nil
```

Comment:
Shouldn't this just be a tx.Get if we only care about the first row returned?

Reply:
Code reverted to what was originally there before this PR now that there's no error to return.